setup.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+from __future__ import absolute_import, print_function
+import os
+
+from setuptools import setup, find_packages
+
+NAME = 'ft'
+VERSION = '1.0.0'
```

Comment:
I'd prefer to start with `0.2` because there might be some breaking changes until we hit `1.0`.

Reply:
OK. will move this to ft.__version__